class-sc-wp-updates-notifier.php

```diff
@@ -534,15 +579,22 @@ private function sort_by_interval( $a, $b ) {
 		 * much straight forward use of the WordPress Settings API.
 		 */
 		public function admin_settings_menu() {
-			$page = add_options_page( 'Updates Notifier', 'Updates Notifier', 'manage_options', 'wp-updates-notifier', array( $this, 'settings_page' ) );
-			add_action( "admin_print_scripts-{$page}", array( $this, 'enqueue_plugin_script' ) );
-		}
-
-		public function enqueue_plugin_script() {
-			wp_enqueue_script( 'wp_updates_monitor_js_function' );
+			add_options_page( 'Updates Notifier', 'Updates Notifier', 'manage_options', 'wp-updates-notifier', array( $this, 'settings_page' ) );
```

Comment:
Should these be translated strings?

```suggestion
			add_options_page( __( 'Updates Notifier', 'wp-updates-notifier' ), __( 'Updates Notifier', 'wp-updates-notifier' ), 'manage_options', 'wp-updates-notifier', array( $this, 'settings_page' ) );
```